packages/synapse-interface/package.json

```diff
@@ -33,7 +33,7 @@
     "@headlessui/react": "1.6.2",
     "@heroicons/react": "1.0.6",
     "@rainbow-me/rainbowkit": "^0.11.0",
-    "@synapsecns/sdk-router": "0.1.27",
+    "@synapsecns/sdk-router": "^0.1.27",
```

Comment:
Changing this breaks the vercel build/deploy action, @abtestingalpha figured that out earlier today, let's keep it pinned, not sure if this is your linter

Reply:
not sure why this is happening either, I thought I had made a fix for this earlier in this branch. might be something to do with my local not resetting when I switch into a diff branch